packages/client/src/lib/api.ts

```
@@ -426,23 +426,23 @@ export const apiClient = {
     updates: { name?: string; participantCentralUserIds?: UUID[]; metadata?: any }
   ): Promise<{ success: boolean; data: MessageChannel }> =>
     fetcher({
-      url: `/messages/central-channels/${channelId}`,
+      url: `/messaging/central-channels/${channelId}`,
       method: 'PATCH',
       body: updates,
     }),
 
   createCentralGroupChannel: (payload: {
     /* ... */
   }): Promise<{ success: boolean; data: MessageChannel }> =>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)